### PR TITLE
Title populate for oneOf when referenced through $ref

### PR DESCRIPTION
**Pull request by abe-101**
Following the suggestion from https://github.com/bhch/django-jsonform/issues/164

For context I have very little understanding of this codebase and very little knowledge in react
I simply want op suggestion fixed

### CODE REVIEW

**Comment by abe-101** [comment]
I just tested out the change locally and it worked like a charm

File: src/ui.js [hunk 1]
```diff
@@ -586,9 +586,7 @@ class OneOfTopLevel extends React.Component {
     }
 
     getOptions = () => {
-        return this.props.args.schema[this.schemaName].map((option, index) => {
-            return {label: option.title || 'Option ' + (index + 1), value: index};
-        });
+        return Array.from(this.props.args.schema[this.schemaName].keys(), (index) => ({label: this.getSchema(index).title || 'Option ' + (index + 1), value: index}));
```
Comment (bhch) [inline src/ui.js:589]: Could you break this up into multiple lines for a bit more readability?

E.g.:

```suggestion
        return Array.from(
            this.props.args.schema[this.schemaName].keys(),
            (index) => ({label: this.getSchema(index).title || 'Option ' + (index + 1), value: index})
        );
```

**Comment by abe-101** [comment]
btw
thank you for the 2 projects - makes dealing with json in django a breeze

**Comment by abe-101** [comment]
Any update with this?
R u waiting on me to merge?

**Comment by bhch** [comment]
Hi, sorry to have kept you waiting.
I've merged this now and will release it in a couple of days.

Thank you for contributing.

**Comment by bhch** [comment]
Released in [django-jsonform v2.23.0](https://github.com/bhch/django-jsonform/releases/tag/v2.23.0).

**Pull request merged**